Kite SDK/PSPrintSDK/OLProduct.h

```diff
@@ -27,5 +27,7 @@
 -(void)setCoverImageToImageView:(UIImageView *)imageView;
 -(void)setProductPhotography:(NSUInteger)i toImageView:(UIImageView *)imageView;
 +(OLProduct *) productWithTemplateId:(NSString *)templateId;
+- (NSDecimalNumber*) decimalNumberUnitCost;
```

Comment:
Very pedantic but how about renaming to "unitCostDecimalNumber" to share prefix of other "unitCost" method? This should hopefully pop up both then when XCode provides autocomplete suggestions?